lib/stateTransition/RawStateTransition.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @typedef RawStateTransition
+ * @property {number} protocolVersion
```

Comment:
Let’s move it to AbstractStateTransition ?

Reply:
Do you mean the contents of the file? Sure

Reply:
Fixed